content/300-guides/200-deployment/201-serverless/400-deploy-to-aws-lambda.mdx

```diff
@@ -1,301 +1,199 @@
-- An [AWS](https://aws.amazon.com/) account and a corresponding [access key](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html) for programmatic access.
-- A [Serverless Framework CLI](https://www.serverless.com/framework/docs/getting-started/) installed. See the note below about [Serverless Framework CLI commands](#serverless-framework-cli-commands).
-- PostgreSQL CLI `psql` installed.
-- Node.js installed.
-
-### Serverless Framework CLI commands
-
-The exact Serverless Framework commands that you must use depend on the way that you install the Serverless Framework CLI.
-
-- If you install the Serverless Framework CLI globally, then you can use the commands exactly as shown in this guide.
-
-- If you install the Serverless Framework CLI in your project, then you must prepend `npx` to the commands in this guide. For example:
-
-  ```terminal wrap
-  npx serverless config credentials --provider aws --key AWS_ACCESS_KEY_ID  --secret AWS_SECRET_ACCESS_KEY
-  ```
-
-## Prisma workflow
+- An existing Serverless Framework project with Prisma.
 
-In this guide, you do the following:
+## Things to know
 
-- Use Prisma and a Prisma schema file (`schema.prisma`) to create a schema for the database
-- Use `prisma db push` to push this Prisma schema to the database
+While Prisma works well with the Serverless Framework "out of the box", there are a few improvements that can be made within your project to ensure a smooth deployment and performance.
 
-In this guide, you use `prisma db push` to push the initial Prisma schema to the database. For a real project, you would typically start with a development database and use `prisma db push` as we suggest here. You can continue with `prisma migrate dev`, which generates a migration script for each change that you make to the Prisma schema. With multiple migration scripts, you build a migration history for your database and that makes it easy to later apply the changes to a production database with `prisma migrate deploy`. For more information, see [Prisma Migrate](/concepts/components/prisma-migrate). <br></br><br></br>If you use Supabase to host your database, then when you use `prisma migrate` you must set up a shadow database. For details, see the [Supabase docs](https://supabase.com/docs/guides/integrations/prisma).
+### Deploy only the required files
 
-## 1. Download the example project
+To reduce your deployment footprint, you can change your deployment process so that you only upload the files that your application needs. The Serverless configuration file `serverless.yml` includes a package pattern that includes only the Prisma engine binary relevant to the Lambda runtime, and excludes the others. This means that when Serverless packages your app for upload, it includes only one binary. This ensures the packaged archive is as small as possible.
```

Comment:
```suggestion
To reduce your deployment footprint, you can change your deployment process so that you only upload the files that your application needs. The Serverless configuration file `serverless.yml` below show a `package` pattern that includes only the Prisma engine file relevant to the Lambda runtime, and excludes the others. This means that when Serverless Framework packages your app for upload, it includes only one engine file. This ensures the packaged archive is as small as possible.
```